certified-connectors/Asite/apiDefinition.swagger.json

```diff
@@ -0,0 +1,769 @@
+{
+	"swagger": "2.0",
+	"info": {
+		"title": "Asite",
```

Comment:
@asitedeveloper Indentation - The formatting of both the swagger and apiProperties.json files is not correct. The properties are not aligned properly and there is longer tab index between the properties. Please refer the certified connectors in the [open source repo](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/certified-connectors/Africa%27s%20Talking%20Airtime/apiDefinition.swagger.json) for reference.